@xen-orchestra/lite/src/components/host/network/HostPifsTable.vue

```diff
@@ -238,6 +244,8 @@ const { visibleColumns, rows } = useTable('pifs', filteredPifs, {
   ],
 })
 
+const { pageRecords: vifRowRecords, paginationBindings } = usePagination('pifRow', rows)
```

Comment:
```suggestion
const { pageRecords: pifsRecords, paginationBindings } = usePagination('pifs', rows)
```

Reply:
Or maybe this naming ? 
```suggestion
const { pageRecords: pifsRecords, paginationBindings } = usePagination('pifsRows', rows)
```
or 
```suggestion
const { pageRecords: pifsRows, paginationBindings } = usePagination('pifsRows', rows)
```

What do you think ? and @OlivierFL  ?